net/local/local_connect.c

```diff
@@ -143,7 +129,7 @@ static int inline local_stream_connect(FAR struct local_conn_s *client,
 
   if (nxsem_get_value(&server->lc_waitsem, &sval) >= 0 && sval < 1)
     {
-      _local_semgive(&server->lc_waitsem);
+      nxsem_post(&server->lc_waitsem);
```

Comment:
why change

Reply:
use nxsem_post & nxmutex_unlock inside

Reply:
but net_lockedwait_uninterruptible doesn't equals to nxsem_post.